README.md

```diff
@@ -45,6 +45,8 @@ If that's not your case and you want to install the necessary backends, follow t
 
 Follow the instructions provided [here](https://docs.bluerobotics.com/ardusub-zola/software/control-station/Cockpit-0.0/advanced-usage/#general) to connect Cockpit to your vehicle.
 
-If you're serving `mavlink2rest` and `mavlink-camera-manager` in the same machine as Cockpit, you can just use `127.0.0.1` as your "Global vehicle address" and everything should work.
+If you're running a BlueOS instance, just put the IP address of it in "Global vehicle address" and click the "apply" button.
 
+If you're serving `mavlink2rest` in the same machine as Cockpit, activate the checkmark for "Mavlink2Rest connection" and use `ws://0.0.0.0:8088/v1/ws/mavlink` there (if you're running a mavlink2rest version before 0.11.15, omit the `/v1` in the URL).
 
+If you're serving `mavlink-camera-manager` in the same machine as Cockpit, activate the checkmark for "WebRTC connection" and use `ws://0.0.0.0:6020` there.
```

Comment:
```suggestion
If you're serving `mavlink-camera-manager` in the same machine as Cockpit, activate the checkmark for "WebRTC connection" and use `ws://127.0.0.1:6020` there.
```

Reply:
Actually it worked for Don using 0.0.0.0. Have you tested with 127.0.0.1? If so I can't change.

Reply:
it works, but it doesn't mean it is right. if they are on the same machine, 127.0.0.1 will work.

Reply:
Yeah yeah, you're right. 0.0.0.0 usually just means "anyone" for server side. Will change.

Reply:
Solved. Will merge.